client/web/privacy/src/components/Common/EditorToolbar.vue

```diff
@@ -38,16 +38,14 @@
         <template v-if="deleteShow">
           <c-input-confirm
             v-if="deleteConfirm"
-            :disabled="deleteDisabled || processing"
-            :borderless="false"
+            :processing="processingConfirm"
```

Comment:
the disabled should be same as before, since it should be disabled if something else is processing or if delete is disabled